sos/report/plugins/crio.py

```diff
@@ -50,6 +50,12 @@ def setup(self):
             "crio config"
         ])
 
+        result = self.exec_cmd("kill -USR1 $(pidof crio)")
+        if result['status'] == 0:
```

Comment:
Are you sure the `$(..)` will be properly evaluated? As when I test it on a mocked `crio` command, I always get `result` as:

    {'status': 1, 'output': 'kill: cannot find process "$(pidof"\nkill: cannot find process "crio)"\n', 'truncated': False}

Reply:
Nack to bash-foo in plugins. Get the pid first, and if one exists, make another call to `kill`.

Reply:
makes sense, updated